data/translations.json

```diff
@@ -1,644 +1,768 @@
-      "VulnerabilitiesItemsProductStatusUnderInvestigationLabel": "Untersuchung läuft",
+      "VulnerabilitiesItemsProductStatusUnderInvestigationTitle": "Untersuchung läuft",
       "VulnerabilitiesItemsProductStatusUnderInvestigationDescription": "Es ist noch nicht bekannt, ob diese Versionen von der Sicherheitslücke betroffen sind oder nicht. Es wird jedoch noch untersucht - das Ergebnis wird in einer späteren Version des Dokuments mitgeteilt.",
-      "VulnerabilitiesItemsReferencesLabel": "Referenzen auf Schwachstellen",
+      "VulnerabilitiesItemsReferencesTitle": "Referenzen auf Schwachstellen",
       "VulnerabilitiesItemsReferencesDescription": "Enthält eine Liste von Referenzen, die mit dieser Schwachstellenposition verbunden sind.",
-      "VulnerabilitiesItemsReleaseDateLabel": "Datum der Veröffentlichung",
+      "VulnerabilitiesItemsReleaseDateTitle": "Datum der Veröffentlichung",
       "VulnerabilitiesItemsReleaseDateDescription": "Enthält das Datum und die Uhrzeit, zu der die Sicherheitslücke ursprünglich in die freie Wildbahn gebracht wurde.",
-      "VulnerabilitiesItemsRemediationsLabel": "Liste der Abhilfemaßnahmen",
+      "VulnerabilitiesItemsRemediationsTitle": "Liste der Abhilfemaßnahmen",
       "VulnerabilitiesItemsRemediationsDescription": "Enthält eine Liste von Abhilfemaßnahmen.",
-      "VulnerabilitiesItemsRemediationsItemsLabel": "Abhilfemaßnahme",
+      "VulnerabilitiesItemsRemediationsItemsTitle": "Abhilfemaßnahme",
       "VulnerabilitiesItemsRemediationsItemsDescription": "Gibt Details an, wie eine Schwachstelle behandelt (und vermutlich behoben) werden soll.",
-      "VulnerabilitiesItemsRemediationsItemsCategoryLabel": "Kategorie der Abhilfemaßnahme",
+      "VulnerabilitiesItemsRemediationsItemsCategoryTitle": "Kategorie der Abhilfemaßnahme",
       "VulnerabilitiesItemsRemediationsItemsCategoryDescription": "Gibt die Kategorie an, zu der diese Abhilfemaßnahme gehört.",
-      "VulnerabilitiesItemsRemediationsItemsDateLabel": "Datum der Abhilfemaßnahme",
+      "VulnerabilitiesItemsRemediationsItemsDateTitle": "Datum der Abhilfemaßnahme",
       "VulnerabilitiesItemsRemediationsItemsDateDescription": "Enthält das Datum, ab dem die Abhilfemaßnahme verfügbar ist.",
-      "VulnerabilitiesItemsRemediationsItemsDetailsLabel": "Einzelheiten der Abhilfemaßnahme",
+      "VulnerabilitiesItemsRemediationsItemsDetailsTitle": "Einzelheiten der Abhilfemaßnahme",
       "VulnerabilitiesItemsRemediationsItemsDetailsDescription": "Enthält eine ausführliche, für den Menschen verständliche Erläuterung der Abhilfemaßnahmen.",
-      "VulnerabilitiesItemsRemediationsItemsEntitlementsLabel": "Verzeichnis der Berechtigungen",
+      "VulnerabilitiesItemsRemediationsItemsEntitlementsTitle": "Verzeichnis der Berechtigungen",
```

Comment:
Rephrase

Reply:
Updated, I guess "Verzeichnis" wasn't a very appropriate translation, or did you mean to add more?

Reply:
Yes. "Liste" would be ok, I guess.